lib/util/cache/package/decorator.ts

```diff
@@ -0,0 +1,126 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import * as packageCache from '.';
+
+type Handler<T> = (parameters: DecoratorParameters<T>) => Promise<unknown>;
+type Method<T> = (this: T, ...args: any[]) => any;
```

Comment:
can we use `unknown` as return value here?

Reply:
Unfortunately, it doesn't appear so. Changing this to `unknown` gives me the error

```
Type 'unknown' is not assignable to type 'Promise<number>'
```